CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.1.0 [2021-08-04]
+
+_What's new?_
+
+_Bug fixes_
+
+- Fixed: Restrict get api call only to root compartment and one zone per region ([#264](https://github.com/turbot/steampipe-plugin-oci/pull/264))
```

Comment:
```suggestion
- Fixed: Restrict get API calls to the root compartment and one zone per region in several core and file storage tables ([#264](https://github.com/turbot/steampipe-plugin-oci/pull/264))
```

We can add a bit more information here to let users know what types of tables are affected. Also `API` should be capitalized.